forge-gui/res/cardsfolder/l/lim_dul_the_necromancer.txt

```diff
@@ -1,4 +1,4 @@
-Name:Lim-Dul the Necromancer
+Name:Lim-Dûl the Necromancer
```

Comment:
So weird. All of the other Lim-Dul names had been normalized. Not sure how this one ended up with the accent marks.

Reply:
This is on purpose

i want the Non Normalized Names be back on the Cards
while the Normalized Name is handled by the CardDb

Reply:
No I know. I just meant why were Lim-Dul's Vault and other cards using the normalized version. I like just converting the standard name down to normalized for storage/lookups.

Reply:
probably because our system couldn't handle the û vocals.

this MR is for changing that back